frontend/src/charts/utils.ts

```diff
@@ -189,24 +195,28 @@ export function createTitles({
       chartTitle = population
         ? [
             `Population vs distrbution of`,
-            `total ${metric}`,
+            `total ${metric.toLocaleLowerCase()}`,
```

Comment:
curious about this, was it an oversight from before?

Reply:
Yeah. I think Mahia suggested that we have sentencing casing for our titles and I just noticed that I missed this from before.

Reply:
Cool